### DISCUSSION TITLE
Git Clone in terminal Git Bash

### DISCUSSION THREAD

**Discussion by Marindala** [community/community#121720] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

When I clone a repository in my gitbash terminal it generates these symbols ^[[200~https://github.com/Marindala/Manipulacion-DOM.git~
I am looking for a solution

**Comment by Marindala** [comment]
I was able to test what happens when I copy from click in  Copy url to clipboard

**Reply by gurjeetsinghvirdee** [reply]
it happens sometimes when you try to paste it by right click 
just remove the `^[[200` just after pasting the link, use your **left arrow** and remove it 

Restart your system it will fix it

**Reply by Marindala** [reply]
yes, go through that, I just want to solve it

**Comment by smilefox-lab** [comment, marked as answer]
It seems like you are encountering special characters or escape sequences when cloning a repository using Git Bash.

**Reply by Marindala** [reply]
Yes, can it be solved from the Git Bash configuration?

**Reply by smilefox-lab** [reply]
Yes, to resolve this problem, you can try the following steps:

Verify your Git configurations by running the following command in your Git Bash terminal:

Check git configurations:
git config --list

Reset default branch name:
git config --global init.defaultBranch main

Clear git cache:
git config --global core.compression 0

 If you have set up an SSH key for GitHub authentication, try using the SSH URL instead of HTTPS to clone the repository. 

Check if any special characters are set to display in the terminal.

Let me know if you continue to face any problems, and I'll do my best to assist you further!

**Reply by smilefox-lab** [reply]
try?

**Reply by Marindala** [reply]
Yes! thank you very much!!

**Comment by gurjeetsinghvirdee** [comment]
I think your solution is here
https://superuser.com/questions/1532688/pasting-required-text-into-terminal-emulator-results-in-200required-text

Go through with this

**Reply by Marindala** [reply]
I will study it. Thank you very much!
